src/tests/class_tests/openms/source/MRMIonSeries_test.cpp

```diff
@@ -100,16 +100,49 @@ START_SECTION((boost::unordered_map<String, double> MRMIonSeries::getIonSeries(A
   TEST_REAL_SIMILAR(ionseries1["y7^3"], 277.80799942)
 
   // Enable neutral losses
-  MRMIonSeries::IonSeries ionseries2 = mrmis.getIonSeries(AASequence::fromString(String("PEPTIDEK")), 3, fragment_types, fragment_charges, true);
+  MRMIonSeries::IonSeries ionseries2 = mrmis.getIonSeries(AASequence::fromString(String("PEPTIDEK")), 3, fragment_types, fragment_charges, true, true);
```

Comment:
can you also test the combination `true, false` here? to showcase what the difference is of the new paramter